code/game/gamemodes/clock_cult/clock_items/judicial_visor.dm

```diff
@@ -168,8 +168,7 @@
 	for(var/mob/living/carbon/C in range(1, src))
 		var/datum/status_effect/belligerent/B = C.apply_status_effect(STATUS_EFFECT_BELLIGERENT)
 		if(!QDELETED(B))
-			B.duration = world.time + 30
-			C.Knockdown(5) //knocks down for half a second if affected
```

Comment:
Keep this, maybe raise it to 8

Reply:
I'm still use to the tick stun system but consider me dubious about the accuracy of these when they're under one tick in length.

Reply:
They're accurate. I made sure of it rather than them being weird-life-tick based.